src/modules/GroupChannel/components/SuggestedReplies/index.tsx

```diff
@@ -24,21 +194,27 @@ const SuggestedReplies = ({ replyOptions, onSendMessage }: SuggestedRepliesProps
     return null;
   }
 
+  const children = replyOptions.map((option: string, index: number) => {
+    return (
+      <div
+        className="sendbird-suggested-replies__option"
+        id={option}
+        key={index + option}
+        onClick={(e) => onClickReply(e, option)}
+      >
+        {option}
+      </div>
+    );
+  });
+
   return (
-    <div className="sendbird-suggested-replies">
-      {replyOptions.map((option: string, index: number) => {
-        return (
-          <div
-            className="sendbird-suggested-replies__option"
-            id={option}
-            key={index + option}
-            onClick={(e) => onClickReply(e, option)}
-          >
-            {option}
-          </div>
-        );
-      })}
-    </div>
+    type === 'vertical'
+      ? <div className="sendbird-suggested-replies vertical">
+        {children}
+      </div>
+      : <DraggableSuggestedReplies>
+        {children}
+      </DraggableSuggestedReplies>
```

Comment:
Wouldn't using a `div` with `overflow: scroll-y` suffice?

Reply:
we had to go with carousel-like because of scroll not blocking x-scroll when y-scroll is active and vice versa

Reply:
Thank you @bang9 for confirming overflow-x: scroll does block y scroll when scrolling x direction. I will change to using overflow-x because of the performance and that it ensures blocking y direction.